editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -692,7 +692,7 @@ function useSelectOrLiveModeSelectAndHover(
   const { onMouseMove: innerOnMouseMove } = useHighlightCallbacks(
     active,
     cmdPressed,
-    false,
+    true,
```

Comment:
Looks like this is now `true` everywhere, so we should just remove that param

Reply:
Made that change in 51d530ebf.